test/conformance/test_suite_test.go

```diff
@@ -5,32 +5,45 @@ package test_test
 
 import (
 	"flag"
+	"os"
+	"strings"
 	"testing"
 
 	. "github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/reporters"
 	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
 
 	_ "github.com/openshift/ptp-operator/test/conformance/ptp"
+	"github.com/openshift/ptp-operator/test/utils/clean"
 	testclient "github.com/openshift/ptp-operator/test/utils/client"
 )
 
 // TODO: we should refactor tests to use client from controller-runtime package
 // see - https://github.com/openshift/cluster-api-actuator-pkg/blob/master/pkg/e2e/framework/framework.go
 
 var junitPath *string
+var DeletePtpConfig bool
 
 func init() {
 	junitPath = flag.String("junit", "junit.xml", "the path for the junit format report")
 }
 
+func InitDeletePtpConfig() {
+	value, isSet := os.LookupEnv("KEEP_PTPCONFIG")
```

Comment:
if value, isSet := os.LookupEnv("KEEP_PTPCONFIG");isSet{

Reply:
I am doing more test than just checking isSet, this is why I chose to not use a one liner

Reply:
Right, I see that now.